net-p2p/nicotineplus/nicotineplus-3.3.10.recipe

```diff
@@ -0,0 +1,103 @@
+SUMMARY="A graphical client for the Soulseek P2P network"
+DESCRIPTION="Nicotine+ aims to be a lightweight, pleasant, free and open source (FOSS) \
+alternative to the official Soulseek client, while also providing a comprehensive set of features."
+HOMEPAGE="https://nicotine-plus.org/"
+COPYRIGHT="2001–2025 Nicotine+, Nicotine and PySoulSeek Contributors"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/nicotine-plus/nicotine-plus/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="3917ebc562f2d6a6b26b3d815d7cbdf1d11c058d994b1f47794bbb850489b35e"
+SOURCE_DIR="nicotine-plus-$portVersion"
+ADDITIONAL_FILES="nicotineplus.rdef.in"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	$portName = $portVersion
+	cmd:nicotine
+	"
+REQUIRES="
+	haiku
+	lib:libgtk_3
```

Comment:
this probably won't work on x86. That's OK, maybe switch the package to x86_64 only, with a comment in the recipe.

Reply:
Thanks for all the suggestions. pygobject doesn't depend on GTK as I can see from the recipe, so the dependency on libgtk will have to stay.

@korli you mentioned this probably won't work on x86. Since the GTK recipe mentions

`ARCHITECTURES="all !x86_gcc2"
SECONDARY_ARCHITECTURES="x86"`

Shouldn't I use this same architectures on this recipe or is there a different reason?